plone/app/blocks/layoutbehavior.py

```diff
@@ -436,7 +444,7 @@ def __len__(self):
 )
 def getLayoutAwareSiteLayout(content):
     lookup = ILayoutAware(content)
-    return lookup.content_layout()
+    return lookup.site_layout()
```

Comment:
Well, don't give too much on the names, it was not always what it seemed to be. This is the deprecated function, so keep it as is, I am sure I had a reason to make it this way. Better do not use it.